SRM 325/RGBStreet.cpp

```diff
@@ -0,0 +1,192 @@
+#include <cstdio>
+#include <cmath>
+#include <cstring>
+#include <ctime>
+#include <iostream>
+#include <algorithm>
+#include <set>
+#include <vector>
+#include <sstream>
+#include <typeinfo>
+#include <fstream>
+
+using namespace std;
+
+int dp[100][100];
+
+class RGBStreet {
+    public:
+
+    int ConvertStringToInt(string s) {
```

Comment:
in the camel case first letter is always small. it should be like `convertStringToInt `